### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by Nkaleth**
# Learning objectives of this PR:

- Use database transactions.
- Modify and delete data in SQL.
- Prepare complex queries that answer analytical questions.

# Screenshoots

# transactions 

3a
3b

# queries 

4a
4b
4c
4d
4e
4f

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -4,8 +4,9 @@ CREATE TABLE animals (
     id INT GENERATED ALWAYS AS IDENTITY,
 	name VARCHAR(15) NOT NULL,
 	date_of_birth DATE NOT NULL,
-    escape_attempts INT NOT NULL, 
+  escape_attempts INT NOT NULL, 
 	neutered BOOLEAN NOT NULL,
 	weight_kg DECIMAL NOT NULL,
+	species VARCHAR(15),
```
Comment (CollinsTatang) [inline schema.sql]: - [x] Please use `ALTER TABLE` to add new column `species` to the animals table.
**Take a Look**
```sql
ALTER TABLE animals ADD species varchar(255);
```

File: queries.sql [hunk 2]
```diff
@@ -8,3 +8,105 @@ SELECT name, escape_attempts From animals WHERE weight_kg > 10.5;
...
+
+/* What is the average weight of animals? */
+
+SELECT AVG(weight_kg) FROM animals;
+
+/* Who escapes the most, neutered or not neutered animals? */
+
+SELECT neutered, SUM(escape_attempts) as total_escape_attempts
+FROM animals
+GROUP BY neutered;
```
Comment (CollinsTatang) [inline queries.sql:99]: **Optional**
- [x] Please you can better write the query for who escapes the most, as seen below
**Take a Look**
```sql
 SELECT neutered, AVG(escape_attempts) FROM animals
   GROUP BY neutered;
```
Or

```sql
SELECT neutered, MAX(escape_attempts) FROM animals
  GROUP BY neutered;
```

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Nkaleth, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @Nkaleth, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**